app-text/elforkane/elforkane-1.2.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="The electronic Holy Quran browser"
+DESCRIPTION="Electronic Holy Quran browser and reciter. \
+"
+COPYRIGHT="1999-2019 Abou Zakaria"
+HOMEPAGE="https://github.com/zakariakov/elforkane"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/zakariakov/elforkane/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="25f5352472e99e6dd018f6003408a793ee85f140a381d58b7de463e9b0bc2003"
+
+SOURCE_FILENAME="elforkane-$portVersion.tar.gz"
+SOURCE_DIR="elforkane-$portVersion"
+ADDITIONAL_FILES="elforkane.rdef.in"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	elforkane$secondaryArchSuffix = $portVersion
+	app:Elforkane = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	ayat_tarajem_data
+	ayat_tafasir_data
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	cmd:unzip
```

Comment:
Is it required?

Reply:
..after recheck it is not.. removed.. thanks @extrowerk